styles.css

```diff
@@ -2,85 +2,120 @@
   box-sizing: border-box;
 }
 
+button {
+  border: 0;
+  padding-right: 0px;
+  padding-left: 0px;
+  outline: 0;
+  cursor: pointer;
+  font-size: inherit;
+  border: 1px solid #000000;
+  padding-top: 25px;
+  padding-bottom: 25px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
 .title {
   font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
 }
 
-.calc-background {
-  width: 400px;
-  height: 600px;
-  background-color: #000000;
+.buttons-container{
   display: flex;
-  flex-direction: column;
-  justify-content: space-evenly;
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 40px;
 }
 
-.calc-row {
-  display: flex;
-  justify-content: space-around;
-  height: 100px;
+.inputs{
+  flex-basis: 75%;
 }
 
-.answer-number {
-  margin: 0px;
+.numbers {
   display: flex;
-  align-items: center;
-  font-family: "Courier New", Courier, monospace;
-  color: #ffefd5;
-  width: 90%;
-  height: 99%;
-  justify-content: flex-end;
+  flex-wrap: wrap;
 }
 
-.quarter-button {
-  margin: 0px;
-  display: flex;
-  width: 24.5%;
-  height: 99%;
-  justify-content: center;
-  align-items: center;
+.numbers li {
+  flex-basis: 33.33%;
+}
+.numbers li:last-child {
+  flex-basis: 100%;
 }
 
-.half-button {
-  margin: 0px;
+.operations {
+  flex-basis: 25%;
   display: flex;
-  width: 49.5%;
-  height: 99%;
-  justify-content: center;
-  align-items: center;
+  flex-direction: column;
 }
 
-.large-button {
-  margin: 0px;
+.clear-buttons {
   display: flex;
-  width: 74.5%;
-  height: 99%;
-  justify-content: center;
-  align-items: center;
+  flex-wrap: wrap;
 }
 
-.grey-button {
+.cancel-button {
+  flex-basis: 66.66%;
+}
+
+.back-button {
+  flex-basis: 33.33%;
+}
+
+.numbers button, .operations button {
+  width: 100%;
+}
+
+.inputs button {
   background-color: #808080;
 }
 
-.orange-button {
+.operations button{
   background-color: #ffa500;
 }
 
-.grey-button:hover {
+.inputs button:hover {
   background-color: #9b9b9b;
 }
 
-.orange-button:hover {
+.operations button:hover{
   background-color: #ffb62d;
 }
 
-.grey-button:active {
+
+.inputs button:active {
   background-color: #575757;
 }
 
-.orange-button:active {
+.operations button:active {
   background-color: #ca8300;
 }
+
+.calculator {
+  width: 400px;
+  height: 600px;
+  background-color: #000000;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 40px;
+}
+
+.calc-row {
```

Comment:
This is used only once and it contains only 1 child element. How could you make the answer row look like it does without using flex?